### PR TITLE
Add referral information page

### PR DESCRIPTION
**Pull request by wschaefer**
- add phone number to organizations table

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -111,6 +111,7 @@
     t.integer "aid_applications_count", default: 0, null: false
     t.integer "total_payment_cards_count", default: 0, null: false
     t.string "county_names", default: [], array: true
+    t.string "phone_number"
```
Comment (bensheldon) [inline db/schema.rb]: It looks like one org in the spreadsheet has 2 numbers. Could we make this more of a free text field like "contact information" and expect the person inputting the data to format it?

**Review by bensheldon — CHANGES_REQUESTED** [review]
I think the phone number formatting is too strict for the actual data.

File: db/schema.rb (hunk 1 above)
Comment (wschaefer) [reply db/schema.rb]: Good call! Thanks, I'll do that.

**Review by bensheldon — APPROVED** [review]
👍

**Comment by wschaefer** [comment]
In case I am not the person who merges this: this PR should be Squash and Merge because I create a migration in one commit and modify it in another. Also, after merging and deploying we will need to manually enter the contact_information for each organization on production.

**Pull request merged**